app/ssttparrangement/ArrangementSubmissionStatus.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ssttparrangement
+
+import enumeratum._
+
+import scala.collection.immutable
+
+sealed trait ArrangementSubmissionStatus extends EnumEntry
+
+object ArrangementSubmissionStatus extends Enum[ArrangementSubmissionStatus]
+  with PlayInsensitiveJsonEnum[ArrangementSubmissionStatus] {
+
+  val values: immutable.IndexedSeq[ArrangementSubmissionStatus] = findValues
+
+  case object Success extends ArrangementSubmissionStatus
+  case object NotSuccessfulQueuedForRetry extends ArrangementSubmissionStatus
```

Comment:
NotSuccessfulQueuedForRetry is probably a bad name as it may not have been added to the queue for a valid reason. `NotQueuedForRetry` may be closer.

Reply:
My choice of language needs improvement, clearly.

This status is supposed to say: "DES call was not successful the first time, so it has been queued for retry". I was going for: " Not successful - queued for retry", as opposed to straight up success, and permanent failure (failed and not added to queue.

Do you have suggestions for a better phrase?

How about:
- just QueuedForRetry
- DESServerErrorQueuedForRetry
?

Reply:
hi :-) 
Its more to do with why it wasn't added to the queue. It DES/ETMP replies with "Invalid value in field X" so calling DES/ETMP again will only come back with the same answer (we hope) and so is not added to the queue. If it's a network problem or something that will work again in X minutes time then it's added to the queue. 

Is DESServerErrorQueuedForRetry explaining that it was a network error and didn't get to DES/ETMP, or that it got to DES/ETMP and was invalid?

Reply:
As discussed, updated to just `QueuedForRetry`